### ISSUE TITLE
Please add Version header

### ISSUE THREAD

**Issue by syohex** [randomphrase/company-c-headers#8]
All Emacs package should have `Version` header. Package installation by Cask may be failed if there is no Version header.

See also
- http://www.gnu.org/software/emacs/manual/html_node/elisp/Packaging-Basics.html#Packaging-Basics

**Comment by randomphrase** [comment]
OK that's odd - I use company-c-headers with Cask all the time, and it works fine. Can you please provide an error message?

To date, I have deliberately not added a version number because I haven't had sufficient feedback on whether this code is sufficiently mature to call a 'release'.

**Comment by syohex** [comment]
Sorry I cannot reproduce my issue. I tested [company-arduino](https://github.com/yuutayamada/company-arduino) with Cask yesterday, then I got following error.

```
Some dependencies were not available: company-arduino.el
```

I got same error with minimized code as below, so I assumed this caused by that `company-c-headers` does not have `Version` header.
```
;; Package-Requires: ((company-c-headers "20140930"))
```

But I cannot get this issue today. So it may be network issue.
Sorry for my trouble.
